package.json

```diff
@@ -75,7 +75,7 @@
     "@fontsource/noto-sans-sc": "4.5.12",
     "@fontsource/noto-sans-tc": "4.5.12",
     "@jellyfin/libass-wasm": "4.1.1",
-    "@jellyfin/sdk": "0.7.0",
+    "@jellyfin/sdk": "unstable",
```

Comment:
I'm not sure how renovate will handle this, but I guess the only way to find out is to try it and then adjust the rules as needed. :sweat_smile: